include/tvm/meta_schedule/search_strategy.h

```diff
@@ -101,6 +131,9 @@ class MeasureCandidate : public runtime::ObjectRef {
 */
 class SearchStrategyNode : public runtime::Object {
  public:
+  /*! \brief Tuning statistics time usage counter. */
+  static TuningStatistics time_counter;
```

Comment:
Let's find ways other than static members - we usually avoid doing so for various reasons :-)